Automattic/jetpack
# My Plan: Show available storage in description for new Backup/Security plans

Resolves `1200412004370260-as-1201289205953291`.

#### Changes proposed in this Pull Request:

* On the **My Plan** page, mention the available backup storage for new Backup and Security plans in the plan card's description.
* Update Backup and Security product names in the Dev Tools card to explicitly list storage amounts.

#### Jetpack product discussion

`p1HpG7-dpL-p2`

#### Does this pull request change what data or activity we track or use?

No change 👍 

#### Testing instructions

1. Visit the **Jetpack > Dashboard > My Plan** page.
2. Open the Dev Tools panel by clicking the **Dev Tools** link in the page footer.
3. Select any of the new Backup or Security plans to see that plan's card.
4. Verify that for each new plan, you see a line in the description stating "You have XX of storage space," where the storage amount is correctly reflected for the selected plan. The storage amount should also be bolded (see reference screenshots).

#### Reference screenshots

<img width="375" alt="Screen Shot 2021-11-09 at 08 08 20" src="https://user-images.githubusercontent.com/670067/140947663-8e6f04bf-bbd6-43bf-9097-c47ad52ff501.png"> <img width="375" alt="Screen Shot 2021-11-09 at 08 08 31" src="https://user-images.githubusercontent.com/670067/140947665-03ab856f-d4b9-4112-b7f3-925f8ef05856.png">

<img width="375" alt="Screen Shot 2021-11-09 at 08 08 41" src="https://user-images.githubusercontent.com/670067/140947667-a1f55741-e958-448c-b995-7c4744d48e3e.png"> <img width="375" alt="Screen Shot 2021-11-09 at 08 08 54" src="https://user-images.githubusercontent.com/670067/140947669-ed7dcfbd-9f16-435c-a970-23b28be1cf5b.png">

<img width="172" alt="image" src="https://user-images.githubusercontent.com/670067/140947954-5ed23db3-c2fa-48bf-a420-d274489cafdd.png">
